metrics/glossary.md

```diff
@@ -0,0 +1,11 @@
+## Glossary
+
+- Metrics - Arbitrary measurements that represents current or historical data from a Service, App or User. Some examples:
```

Comment:
Metrics - are usually connected to some changing values: like performance metrics etc.
In this case statistics may be better name.